packages/kmath/src/angles.ts

```diff
@@ -8,6 +8,12 @@ export function convertDegreesToRadians(degrees: number): number {
     return (degrees / 180) * Math.PI;
 }
 
+export function convertRadiansToDegrees(radians: number): number {
```

Comment:
No action needed, but I was musing on naming in this file.  We have `calculate`, `convert`, `get`, and bare action naming. 

I think I'd be inclined to remove the prefixes as they don't really add much value (is there a difference between "calculating" or "converting" or "getting" a value given another value? It's all a transformation/calculation. 

That would leave us with an angles API like this: 

* `degreesToRadians(degrees: number): number`
* `radiansToDegrees(radians: number): number`
* `angleToDegrees([x, y]: Coord): number`
* `angleFromVertex...`
* `clockwiseAngle...`

Not asking you to change anything, but I feel having a consistent naming convention for these types of things is useful. :) Dig in if you feel motivated and agree though!